llms/tei/text_embeddings_inference..go

```diff
@@ -0,0 +1,43 @@
+package tei
```

Comment:
Maybe it's worth adding a comment here explaining this package? Specifically, it's not a usual LangChainGo LLM, but only a shell for invoking an embeddings API.